### PR TITLE
update babel to version 7

### LINKED ISSUE #21: update babel to version 7
Hello, @aichbauer. This module is useful for me to create REST API back-end. Now, I am using babel version 7 in my project. I have used this package for mapping routes easily. Unfortunately, I have this error message after updating to babel version 7.

```bash
Error: Requires Babel "^7.0.0-0", but was loaded with "6.26.3". If you are sure you have a compatible version of @babel/core, it is likely that something in your build process is loading the wrong version. Inspect the stack trace of this error to look for the first entry that doesn't mention "@babel/core" or "babel-core" to see what is calling Babel.
    at throwVersionError (${MY_PROJECT_DIR}\node_modules\@babel\helper-plugin-utils\lib\index.js:65:11)
    at Object.assertVersion (${MY_PROJECT_DIR}\node_modules\@babel\helper-plugin-utils\lib\index.js:13:11)
    at api (${MY_PROJECT_DIR}\node_modules\@babel\plugin-proposal-function-sent\lib\index.js:51:7)
    at ${MY_PROJECT_DIR}\node_modules\@babel\helper-plugin-utils\lib\index.js:19:12
    at Function.memoisePluginContainer (${MY_PROJECT_DIR}\node_modules\babel-core\lib\transformation\file\options\option-manager.js:113:13)
    at Function.normalisePlugin (${MY_PROJECT_DIR}\node_modules\babel-core\lib\transformation\file\options\option-manager.js:146:32)
    at ${MY_PROJECT_DIR}\node_modules\babel-core\lib\transformation\file\options\option-manager.js:184:30
    at Array.map (<anonymous>)
    at Function.normalisePlugins (${MY_PROJECT_DIR}\node_modules\babel-core\lib\transformation\file\options\option-manager.js:158:20)
    at OptionManager.mergeOptions (${MY_PROJECT_DIR}\node_modules\babel-core\lib\transformation\file\options\option-manager.js:234:36)
    at OptionManager.init (${MY_PROJECT_DIR}\node_modules\babel-core\lib\transformation\file\options\option-manager.js:368:12)
    at compile (${MY_PROJECT_DIR}\node_modules\babel-register\lib\node.js:103:45)
    at loader (${MY_PROJECT_DIR}\node_modules\babel-register\lib\node.js:144:14)
    at Object.require.extensions.(anonymous function) [as .js] (${MY_PROJECT_DIR}\node_modules\babel-register\lib\node.js:154:7)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at ${MY_PROJECT_DIR}\node_modules\express-routes-mapper\lib\index.js:66:17
    at Array.forEach (<anonymous>)
    at mapRoutes (${MY_PROJECT_DIR}\node_modules\express-routes-mapper\lib\index.js:47:13)
    at Object.<anonymous> (${MY_PROJECT_DIR}\src\index.js:74:59)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Module._compile (${MY_PROJECT_DIR}\node_modules\pirates\lib\index.js:83:24)
```

Can I help you to upgrade this module using babel 7?

### PR DESCRIPTION
**Pull request by achmadk**
Fixed #21 issue.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,18 +32,20 @@
   "author": "Lukas Aichbauer  <l.aichbauer@gmail.com> (https://github.com/aichbauer/)",
   "license": "MIT",
   "dependencies": {
-    "babel-core": "^6.23.1",
-    "babel-plugin-add-module-exports": "^0.2.1",
-    "babel-plugin-transform-runtime": "^6.23.0",
-    "babel-preset-env": "^1.2.1",
-    "babel-register": "^6.23.0",
-    "express": "^4.15.2",
+    "@babel/cli": "^7.0.0",
```
Comment (aichbauer) [inline package.json]: We should move the `@babel/*` dependencies to devdependencies except of `@babel/runtime`.

File: src/index.js [hunk 2]
```diff
@@ -41,20 +41,12 @@ const mapRoutes = (routes, pathToController) => {
...
-    if (requestMethod === 'get') {
-      router.route(myPath).get(contr[controllerMethod]);
-    } else if (requestMethod === 'post') {
-      router.route(myPath).post(contr[controllerMethod]);
-    } else if (requestMethod === 'put') {
-      router.route(myPath).put(contr[controllerMethod]);
-    } else if (requestMethod === 'delete') {
-      router.route(myPath).delete(contr[controllerMethod]);
-    }
+    router.route(myPath)[requestMethod](contr[controllerMethod]);
```
Comment (aichbauer) [inline src/index.js:49]: Nice 👍 I like this approach

**Review by aichbauer — CHANGES_REQUESTED** [review]
Thanks for your contribution... Looks good... Just one little thing.

Also when we want to merge this you need to drop the node version 4 in the travis file.

File: package.json (hunk 1 above)
Comment (achmadk) [reply package.json]: I have tried to move that `@babel/*` dependencies to devDependencies, but it will give an error when execute `npm run test`

Comment (aichbauer) [reply package.json]: Try moving these to devDependencies:

```json
    "@babel/cli": "^7.0.0",
    "@babel/core": "^7.0.0",
    "@babel/plugin-transform-runtime": "^7.0.0",
    "@babel/preset-env": "^7.0.0",
```

and keep this as dependencies:

```json
    "@babel/register": "^7.0.0",
    "@babel/runtime": "^7.0.0",
```

**Comment by aichbauer** [comment]
in the `travis.yml` you need to remove node version 4.

```yml
node_js:
  - 4
  - 6
  - 8
  - 10
```
to
```yml
node_js:
  - 6
  - 8
  - 10
```

**Review by aichbauer — APPROVED** [review]

**Pull request merged**